setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
no way we can get it merged at this point its still in the preview of the API for github. Therefore things can change. Its also a good question to see when it will get merged into the mainline for github's api and then we can try to get the PyGithub merged.